doc/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,320 @@
+:github-address: https://github.com/hazelcast-guides/kubernetes-hpa
+:templates-url: templates:ROOT:page$/
+:hazelcast: Hazelcast IMDG
+:framework: Kubernetes
+
+
+= Kubernetes HPA for Hazelcast cluster
+
+Cloud technologies give you on-demand options so that you can create compute, disk or network resources based on your requirements. When your demand changes, you update your infrastructure by releasing some resources or adding more. That is actually named Manual Scaling based on human intervention. Kubernetes is no different in this particular use case. If you create a service on top of Kubernetes, and you see unexpected popularity than you planned for, you need to scale out your number of Pods to match the traffic coming to your application. Kubernetes has an automated solution for this problem. https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[Horizontal Pod Autoscaler (HPA)] automatically scales the number of pods in Kubernetes based on your metrics selection. You have two options to choose.
+
+* https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/#resource-metrics-pipeline[Resource Metrics]
+* https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/#full-metrics-pipeline[Custom Metrics]
+
+Let's dig into each of them with a supporting example using Hazelcast
```

Comment:
I think we could shorten this paragraph to something like: `This guide will show you how to set up Hazelcast with [Kubernetes Horizontal Pod Autoscaler(HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).`. I don't think this whole description is really needed.